### PR TITLE
refactor(cli): Make some run params reusable

### PR DESCRIPTION
**Pull request by yjhmelody**
Some offchain service maybe need to reuse some components, so they could reuse these params.

--- 
polkadot address: 15ouFh2SHpGbHtDPsJ6cXQfes9Cx1gEFnJJsJVqPGzBSTudr

### CODE REVIEW

File: client/cli/src/params/prometheus_params.rs
Comment (koushiro) [inline client/cli/src/params/prometheus_params.rs:1]: please add the license header

**Comment by yjhmelody** [comment]
@bkchr Need a review. Thx!

File: client/cli/src/params/runtime_params.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	#[arg(long, default_value_t = 2)]
+	pub runtime_cache_size: u8,
+}
+
+fn parse_max_runtime_instances(s: &str) -> Result<usize, String> {
+	let max_runtime_instances = usize::from_str(s)
+		.map_err(|_err| format!("Illegal `--max-runtime-instances` value: {s}"))?;
+
+	if max_runtime_instances > 256 {
+		Err(format!("Illegal `--max-runtime-instances` value: {max_runtime_instances}"))
```
Comment (bkchr) [inline client/cli/src/params/runtime_params.rs:39]:
```suggestion
		Err(format!("Illegal `--max-runtime-instances` value: {max_runtime_instances} is more than the allowed maximum of `256` "))
```

**Review by bkchr — APPROVED** [review]

**Review by michalkucharczyk — APPROVED** [review]

**Pull request merged**